CSA.md

```diff
@@ -124,25 +126,25 @@ During the <span class="orderform_link">Subscription Period</span> and for six m
 
```

Comment:
Update list-item separator to semi-colon instead of comma

```suggestion
12.1 <span class="header_3">Non-Use and Non-Disclosure.</span>  Unless otherwise authorized in the Agreement, Recipient will (a) only use Discloser's Confidential Information to fulfill its obligations or exercise its rights under this Agreement; and (b) not disclose Discloser's Confidential Information to anyone else. In addition, Recipient will protect Discloser's Confidential Information using at least the same protections Recipient uses for its own similar information but no less than a reasonable standard of care.
```